services/frontend/src/scenes/Profile/Profile.scss

```diff
@@ -1,54 +1,69 @@
 @import "styles/_colors.scss";
 @import "styles/_dimens.scss";
+@import "styles/_media_queries.scss";
 @import "styles/_mixins.scss";
 
-.profile {
-    align-items: flex-start;
+.profile-content {
     display: flex;
-    flex-direction: row;
-    height: calc(100% - #{$navbar-height});
-    justify-content: space-between;
-    padding: 0;
+    flex: 1;
+    flex-direction: column;
+    padding: $default-spacing*2;
 
-    .profile-content {
-        align-items: center;
-        display: flex;
-        flex-direction: column;
-        height: 100%;
-        overflow: auto;
-        padding: $default-spacing;
-
-        .profile-content-title {
-            color: $color-neutral-6;
-        }
+    .profile-content-title {
+        color: $color-neutral-6;
+    }
 
-        .profile-content-card {
-            margin: 16px $default-spacing*5 $default-spacing 0px;
-            padding: $default-spacing;
-        }
+    .profile-card:first-child {
+        margin-top:$default-spacing;
     }
 
-    .profile-sidebar {
-        align-items: center;
+    .profile-card {
         display: flex;
-        flex-direction: column;
-        justify-content: center;
-        margin: $default-spacing * 2;        
+        margin:  $default-spacing 0;
+        padding: $default-spacing*2;
+
+        .profile-card-picture {
+            font-size: 30px;
+            height: 64px;
+            width: 64px;
+        }
+        
+        .profile-card-content {
+            margin: 0 $default-spacing*2;
 
-        .profile-list-item {
-            border-left: 3px solid transparent;
-            padding: $default-spacing*0.5 $default-spacing*1.5;
+            .profile-card-title {
+                font-size: $font-size-title;
+                font-weight: bold;
+                margin-bottom: $default-spacing*0.7;

```

Comment:
Just for the record, when I said I like to remove the spaces around the multiplication/division operator, that was with the condition that there was more than one value in a single line (i.e. a margin property with 2 or more values). 

For a single value property, I still put the spaces around the operator. Up to you if you want to fix all of them. Cause it's either all or nothing...

Reply:
Sure, added spaces to the single rules for consistency.